pkg/model/api/v1/dashboard.go

```diff
@@ -27,7 +27,7 @@ import (
 type PanelSpec struct {
 	Display common.Display `json:"display" yaml:"display"`
 	Plugin  common.Plugin  `json:"plugin" yaml:"plugin"`
-	Queries []Query  `json:"queries" yaml:"queries"`
+	Queries []Query        `json:"queries,omitempty" yaml:"queries, omitempty"`
```

Comment:
My go is super weak -- does this exclude queries entirely if they're not set, turn them into an empty array, or some other thing?

Reply:
`omitempty` will remove the `queries` field from object if it's an empty array/undefined. Anyways I removed this because i realized it's not correct. Good call